assets/js/modules/analytics/components/dashboard/DashboardAllTrafficWidgetV2/UserDimensionsPieChart.js

```diff
@@ -83,8 +83,15 @@ export default function UserDimensionsPieChart( { dimensionName, entityURL, sour
 					const { dataTable } = GoogleChart.charts.get( 'user-dimensions-pie-chart' ) || {};
 					if ( dataTable ) {
 						const dimensionValue = dataTable.getValue( row, 0 );
-						setValues( FORM_ALL_TRAFFIC_WIDGET, { dimensionValue } );
+
+						if ( 'Others' !== dimensionValue ) {
+							setValues( FORM_ALL_TRAFFIC_WIDGET, { dimensionValue } );
+						} else {
+							setValues( FORM_ALL_TRAFFIC_WIDGET, { dimensionName, dimensionValue: '' } );
```

Comment:
@asvinb no need to set the `dimensionName` here and in the next `else` section. Please, remove it and leave only the `dimensionValue` property.

Reply:
@eugene-manuilov hmm, it's mentioned in the AC that the dimensionName must be set, unless the AC is outdated?

Reply:
It turned out that it wouldn't work correctly if we set the dimension name here.

Reply:
Gotcha! Updated @eugene-manuilov